packages/rn-tester/js/examples/ScrollView/ScrollViewIndicatorInsetsExample.macos.js

```diff
@@ -1,10 +1,12 @@
 /**
- * Copyright (c) Facebook, Inc. and its affiliates.
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
+// [macOS]
```

Comment:
Should we use block diff here instead?

Reply:
This file is a macOS only file, which I documented as an exception where we can just use a single line diff at the top. We can change that format to a block diff if that makes sense?

Reply:
Sorry for missing that in the doc. I don't have a strong preference, so keeping it as is is fine with me.